frontend/src/components/Notificacao/Form/ExamesImagem/RaioTorax.vue

```diff
@@ -81,6 +87,15 @@ export default {
     updateRaioOutro(raioOutro) {
       this.$emit('update:raioOutro', raioOutro);
     },
+    requiredIfRealizouOutroRaioTorax(value) {
+      if (!this.realizouOutroRaioTorax) {
+        return true;
+      }
+      return required(value, 'O campo é obrigatório.');
```

Comment:
Essa já é a mensagem padrão, não precisa passar nada nesse parâmetro.
```suggestion
      return required(value);
```